packages/web-component/src/lib/types.ts

```diff
@@ -80,6 +80,17 @@ export type DebuggerMessage = {
   description?: string;
 };
 
+export enum MessageLogType {
+  INFO = 'info',
+  ERROR = 'error',
+  WARN = 'warning',
+}
+
+export type MessageLog = {
+  type: MessageLogType;
+  state?: any;
```

Comment:
the term `state` doesn't convey a lot of information with it, we should find out a better naming for it?

Reply:
i agree, however it could be anything and should represent the current "state" of the log. mostly add some details on what is the current state such as steps details and screen details. i would rather keep it vague so we can add whatever we want here